config/distributions/trixie/architectures

```diff
@@ -0,0 +1 @@
+arm64,armhf,amd64
```

Comment:
Perhaps enabling riscv64?
https://release.debian.org/testing/arch_qualify.html

Reply:
Yes, but I heard riscv64 is _in flux_ -- packages are slowly being rebuilt into the new archives.
Will do when I get some riscv64 hw if no-one does it before...

Reply:
Rationale here is - we have some riscv64 activity & images and with Debian, we are currently destined to Sid. Which is just more fragile & worse in this context. Can be added later, once well tested. Its low priority, so not a deal breaker.